liferay-echo/liferay-portal
# LPS-193743 Migrate AUI navigation.js file to js/react

Motivation
=======
The motivation behind this ticket is to migrate from AUI old javascript files to functional/react js components.

Proposed Solution
========
What I did to solve this problem is to change the file structure and to use equivalents methods in new js structure

Steps to Verify
========
1. Go to Content & Data -> Web contents
2. Create a web content
3. Select it by clicking over the checkbox
4. Check that the Expire, Move and Delete action from the navigation bar work correctly
5. Create a new web content folder
6. Drag the web content to the new folder checking is working correctly
7. Drag the web content to the Recicle bin in the Product menu checking it works fine
